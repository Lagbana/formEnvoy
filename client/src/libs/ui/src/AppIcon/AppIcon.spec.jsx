import React from 'react'
import { shallow, configure } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'
import { AppIcon } from '.'

configure({ adapter: new Adapter() })


test('<AppIcon /> renders correctly', () => {
  const component = shallow(<AppIcon name={'move'} />)
  expect(toJson(component)).toMatchSnapshot()
})
