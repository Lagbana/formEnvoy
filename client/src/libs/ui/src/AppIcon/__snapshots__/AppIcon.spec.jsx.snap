// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppIcon /> renders correctly 1`] = `
<Icon
  allowFontScaling={false}
  name="move"
  size={12}
/>
`;
